CMakeLists.txt

```diff
@@ -6,8 +6,28 @@ set(bpftrace_VERSION_MAJOR 0)
 set(bpftrace_VERSION_MINOR 9)
 set(bpftrace_VERSION_PATCH 3)
 
-set(STATIC_LINKING OFF CACHE BOOL "Build bpftrace as a statically linked executable")
 set(WARNINGS_AS_ERRORS OFF CACHE BOOL "Build with -Werror")
+set(STATIC_LINKING OFF CACHE BOOL "Build bpftrace as a statically linked executable")
+set(STATIC_LIBC OFF CACHE BOOL "Attempt to embed libc, only known to work with musl. Has issues with dlopen.")
```

Comment:
I originally worded it EMBED_LIBC, but changed it for that reason - it doesn't do what the other EMBED_ flags do. These flags say "fetch this dependency from source and build it, linking to this instead of the even trying to find the system library", whereas this is about trying to include the already existing static libc libraries. 

I can change it if you still disagree, I felt it needed something to distinguish it from the others which actually trigger new ExternalProjects being built.

Reply:
Sure, that's fine then.

Just in case it wasn't clear, I was also pointing out that I think you forgot to change all occurrences of EMBED_LIBC when you changed it.

Reply:
> Just in case it wasn't clear, I was also pointing out that I think you forgot to change all occurrences of EMBED_LIBC when you changed it.

Yeah i am noticing it now. I have a few branches I am working on (silly me) and i'm pulling in the patches from my learnings in attempting to support these different configurations. I'll grep through the source to check for this before I push again

Reply:
It was masking a failure in the alpine build, thanks for pointing this out. Fixed I believe.